operator/v1/types_network.go

```diff
@@ -325,6 +325,16 @@ type OVNKubernetesConfig struct {
 	// cluster.
 	// +optional
 	IPsecConfig *IPsecConfig `json:"ipsecConfig,omitempty"`
+	// v4JoinSubnet is the v4 join subnet to be used by ovn-kubernetes in case the
```

Comment:
I don't know what a "join subnet" is? Maybe it will be obvious to an administrator but you could add more description about that?

Reply:
Good point. I will add more description about what a join subnet is.